encoding/text/marshal_test.go

```diff
@@ -20,25 +20,34 @@ func TestEncode(t *testing.T) {
 	tests := []struct {
 		constID uint64
 		text    string
+		indent  int
```

Comment:
I'd like to increase the coverage of this new indent mode. Rather than have a separate parameter, let's have each test case include both the indented and not-indented versions.

Reply:
Done.